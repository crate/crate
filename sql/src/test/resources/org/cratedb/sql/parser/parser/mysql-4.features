+INFIX_LOGICAL_OPERATORS
