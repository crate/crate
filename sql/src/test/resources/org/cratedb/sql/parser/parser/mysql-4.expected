org.cratedb.sql.parser.parser.CursorNode@3af42ad0
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@7426dbec
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@23309e87

		[0]:		
		org.cratedb.sql.parser.parser.AllResultColumn@4cf7c31d
		tableName: null
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@1e2acc65

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@bd10a5c
		tableName: t
		updateOrDelete: null
		null
		correlation Name: null
		null
	whereClause: 		
		org.cratedb.sql.parser.parser.OrNode@48ff2413
		operator: or
		methodName: or
		type: null
		leftOperand: 			
			org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@669980d5
			operator: >
			methodName: greaterThan
			type: null
			leftOperand: 				
				org.cratedb.sql.parser.parser.ColumnReference@67d95492
				columnName: x
				tableName: null
				type: null
			rightOperand: 				
				org.cratedb.sql.parser.parser.ColumnReference@6d3f7281
				columnName: y
				tableName: null
				type: null
		rightOperand: 			
			org.cratedb.sql.parser.parser.AndNode@6076ab2f
			operator: and
			methodName: and
			type: null
			leftOperand: 				
				org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@ee0cc23
				operator: <
				methodName: lessThan
				type: null
				leftOperand: 					
					org.cratedb.sql.parser.parser.ColumnReference@487a1576
					columnName: y
					tableName: null
					type: null
				rightOperand: 					
					org.cratedb.sql.parser.parser.ColumnReference@47171b26
					columnName: z
					tableName: null
					type: null
			rightOperand: 				
				org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@3df78040
				operator: <>
				methodName: notEquals
				type: null
				leftOperand: 					
					org.cratedb.sql.parser.parser.ColumnReference@44050988
					columnName: x
					tableName: null
					type: null
				rightOperand: 					
					org.cratedb.sql.parser.parser.ColumnReference@26b4e29b
					columnName: z
					tableName: null
					type: null